samtranslator/plugins/api/implicit_http_api_plugin.py

```diff
@@ -57,6 +57,10 @@ def _process_api_events(self, function, api_events, template, condition=None):
         for logicalId, event in api_events.items():
             # api_events only contains HttpApi events
             event_properties = event.get("Properties", {})
+
+            if event_properties and not isinstance(event_properties, dict):
+                raise InvalidEventException(logicalId, "Event Properties is invalid.")
```

Comment:
should we add a hint of what type it needs to be?

Reply:
That is the part I'm struggling with, what a clear message could be. I don't want to use the python-specific term "dict/dictionary". Do you have a good suggestion?